README.rst

```diff
@@ -416,10 +416,8 @@ Useful Links
 
 - `Pipenv`_: Current reference implementation
 - `pypa/pip#1795`_: Requirements 2.0
-- `Basic Concept Gist`_ (fork of @dstufft's)
 
 .. _`Pipenv`: http://pipenv.org
-.. _`Basic Concept Gist`: https://gist.github.com/kennethreitz/4745d35e57108f5b766b8f6ff396de85
```

Comment:
This is here now: `https://gist.github.com/ken-reitz/4745d35e57108f5b766b8f6ff396de85`

Reply:
Thanks I've amended the commit with the new link